.ci/bump_license_year.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+CURR_YEAR=$(date +"%Y")
+JAVA_FILE=$(find ./src/main -type f -name *.java -print -quit)
+echo "Current year will be taken from $JAVA_FILE"
+PREV_YEAR=$(grep "Copyright" $JAVA_FILE | cut -d " " -f 4 | cut -d '-' -f 2)
+OLD_PATTERN="// Copyright (C) 2001-$PREV_YEAR the original author or authors."
+NEW_PATTERN="// Copyright (C) 2001-$CURR_YEAR the original author or authors."
+find ./src -type f \( -name *.java -o -name *.header -o -name *.g \) \
+  -exec sed -i "s|$OLD_PATTERN|$NEW_PATTERN|g" {} +
+
+echo "Please Verify that output below is ONLY single line:"
```

Comment:
Why can't script help with this and print it?

Reply:
actually it is not good verification, too late to verify, ... I rephrased a message